### PR TITLE
Add ru_RU.lang

### PR DESCRIPTION
**Pull request by Felix14-v2**
Just a localization of my favourite shader pack for my native language :)
Tested it for a while, and so far I haven't found any issues.
[Mistake](https://discord.com/channels/604061216779796492/806083249599807529/874321913151881316) in the descriptions of the SSR and SSGI settings has been taken into account and corrected in the translation.

### CODE REVIEW

**Comment by Eldeston** [comment]
Hello, I have noticed the mistake for SSGI and SSR and will correct. As for the translation, I need another person who can speak in this language that I trust to confirm the translations are correct.

**Review by HyperSoop — APPROVED** [review]

File: shaders/lang/ru_RU.lang [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+    #Камера
+        screen.Camera = Камера
+        option.DOF = Глубина резкости
+        option.DOF.comment = Активирует эффект фокусировки камеры.. Рекомендуется включить сглаживание для лучшего результата..
+        option.DOF_STRENGTH = Изменение резкости
+        option.DOF_STRENGTH.comment = Сила размытия удалённых объектов..
+        option.CHROMATIC_ABERRATION = Хроматическая аберрация
+        option.ABERRATION_PIX_SIZE = Величина аберрации
+        option.VIGNETTE = Виньетироавние
```
Comment (PinkGoosik) [inline shaders/lang/ru_RU.lang:37]: Little typo, should be `Виньетирование`

**Comment by Eldeston** [comment]
Hmmm, after having closer inspection, this format appears to be outdated from 1.3.1. A lot of options have been changed since 1.3.3.

**Comment by PinkGoosik** [comment]
Oh yeah, the format indeed seems to be different from what is [original](https://github.com/Eldeston/Super-Duper-Vanilla/blob/master/shaders/lang/en_US.lang)

**Comment by HyperSoop** [comment]
@Felix14-v2

**Comment by Eldeston** [comment]
Will keep the suggestion open until it is updated to the 1.3.4 format (latest commit) and merge.

**Comment by Eldeston** [comment]
One last note, I added more info for translation in [CONTRIBUTION.md](https://github.com/Eldeston/Super-Duper-Vanilla/blob/master/CONTRIBUTION.md#translation), make sure to read it.

File: shaders/lang/ru_RU.lang (hunk 1 above)
Comment (Felix14-v2) [reply shaders/lang/ru_RU.lang:37]: Oops, i've no idea how my spellchecker missed that, thanks!

**Comment by Felix14-v2** [comment]
> until it is updated to the 1.3.4 format

Done

**Comment by Eldeston** [comment]
Hello, thanks for notifying me. I seem to notice that the lines of code have increased from 226 (latest commit) to 312 (your pull request), did you make sure there aren't any duplicate lines of texts and the file stayed true to the standard text format?

Also please read [CONTRIBUTION.md](https://github.com/Eldeston/Super-Duper-Vanilla/blob/master/CONTRIBUTION.md#translation) before continuing to make changes.

**Comment by Felix14-v2** [comment]
> increased from 226 (my latest commit) to 312

This is because the basic en_US.lang file doesn't contain tooltip translation strings which I also translated.

**Comment by Eldeston** [comment]
Hmmm, I see. Then please double check if your changes stayed true to the original format and lastly add your name in [**TRANSLATORS**](TRANSLATORS.md) under the translators list and include it in your pull request with the language file.

**Comment by Felix14-v2** [comment]
> check if your changes stayed true to the original format

Already checked them, all translated tooltips are displayed ingame correctly (now with Optifine too, despite the lack of space), The construction of sentences also keeps the original format where it is possible.
Also updated TRANSLATORS.md, please, check if I did it the right way.

**Pull request merged**